label_sync/main_test.go

```diff
@@ -45,15 +45,15 @@ func TestSyncLabels(t *testing.T) {
 		},
 		{
 			name: "Duplicate wanted label",
-			config: Configuration{Labels: []Label{
+			config: Configuration{Default: []Label{
```

Comment:
I would expect more/better unit tests here. In particular I do not see where we validate things like:
* What happens if repos foo and bar both define a label L?
* What happens if lable L is defined at both default and repo level?
* Ensure that if label L1 is in default and label LR is defined at repo that the repo includes both L1 and LR
etc

Reply:
I have added some :)